src/Suave/Utils/Aether.fs

```diff
@@ -0,0 +1,223 @@
+﻿module Suave.Utils.Aether
```

Comment:
Similar to the above; might it be worth managing this as a source file dependency via paket?